tests/utils/series.test.ts

```diff
@@ -169,24 +179,54 @@ describe('series options', () => {
     });
 
     describe('dots', () => {
-        const line = {
+        describe('defaults', () => {
+            const line = {
+                name: 'Line',
+                color: COLOR,
+                data: [1, 2, 3],
+            };
+
+            const y = new Yagr(window.document.body, {
+                timeline: [1, 2, 3],
+                chart: {
+                    series: {
+                        type: 'dots',
+                    },
+                },
+                series: [line],
+            });
+
+            it('should setup uPlot fields', () => {
+                expect(configureSeries(y, line, 0)).toHaveProperty('pointsSize', DEFAULT_POINT_SIZE);
+            });
+        });
+    });
+
+    describe('overrides', () => {
+        const POINTS_SIZE = 10;
+
+        const line: RawSerieData<DotsSeriesOptions> = {
+            type: 'dots',
             name: 'Line',
             color: COLOR,
             data: [1, 2, 3],
         };
 
         const y = new Yagr(window.document.body, {
             timeline: [1, 2, 3],
+            series: [line],
             chart: {
                 series: {
                     type: 'dots',
+                    pointsSize: POINTS_SIZE,
                 },
             },
-            series: [line],
         });
 
-        it('should setup uPlot fields', () => {
-            expect(configureSeries(y, line, 0)).toHaveProperty('pointsSize', DEFAULT_POINT_SIZE);
+        it('should override uPlot fields from config.chart.series', () => {
+            // @ts-ignore
```

Comment:
Is it necessary?

Reply:
Yes, fill method presents but it's not typed in uPlot cuz intended to be private, but there's no any public methods to check real fill color in uPlot. I guess it's ok for unit tests

Reply:
> I guess it's ok for unit tests

Sure. I just wanted to clarify. Ship!